src-frontend/app/templates/procedure.hbs

```diff
@@ -1,3 +1,4 @@
 <div class="container-fluid">
+    <a class="add-page" {{action "generateProtocol"}}>Generate</a>
```

Comment:
If you just add `style="btn btn-default"` it will be styled by Bootstrap as a button